oci/namespaces.go

```diff
@@ -4,13 +4,9 @@ import specs "github.com/opencontainers/runtime-spec/specs-go"
 
 // RemoveNamespace removes the `nsType` namespace from OCI spec `s`
 func RemoveNamespace(s *specs.Spec, nsType specs.NamespaceType) {
-	idx := -1
 	for i, n := range s.Linux.Namespaces {
 		if n.Type == nsType {
-			idx = i
+			s.Linux.Namespaces = append(s.Linux.Namespaces[:i], s.Linux.Namespaces[i+1:]...)
```

Comment:
It is not the same, but it seems to me that this PR has the correct behavior. Ping @mlaventure @LK4D4 to confirm though.